src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -298,7 +303,8 @@ public void checkWikiMarkupTools() {
   }
 
   public void checkSymbolsTools() {
-    for (int i = 1; i < 65; i++) {
+    int wikitextShortcutsSymbolsSection = 65; //edit tools 'Symbols' section in MediaWiki:Edittools
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ATrailingCommentCheck)